v21.1/known-limitations.md

```diff
@@ -6,6 +6,159 @@ toc: true
 
 This page describes newly identified limitations in the CockroachDB {{page.release_info.version}} release as well as unresolved limitations identified in earlier releases.
 
+## New limitations
+
+### CockroachDB does not properly optimize some left and inverted joins with inverted indexes
+
```

Comment:
@rytaft

Reply:
just a couple of nits, otherwise LGTM - thanks!